sceneview/src/main/java/io/github/sceneview/node/Node.kt

```diff
@@ -735,14 +736,14 @@ open class Node(
      * @param upDirection The up direction will determine the orientation of the node around the look direction
      * @param smooth Whether the rotation should happen smoothly
      */
-    fun lookTowards(
+    open fun lookTowards(
         lookDirection: Direction,
         upDirection: Direction = Direction(y = 1.0f),
         smooth: Boolean = false
     ) {
         val newQuaternion = lookTowards(
             worldPosition,
-            -lookDirection,
+            -lookDirection, // lookDirection is negated so that the forward direction will be aligned with the lookDirection
```

Comment:
If we put a comment inside the arguments, we can also put it on the preceding line.